test/bioprinttest.c

```diff
@@ -247,3 +253,54 @@ int test_main(int argc, char **argv)
 
     return run_tests(argv[0]);
 }
+
+/*
+ * Replace testutil output routines.  We do this to eliminate possible sources
+ * if BIO error
+ */
+void test_open_streams(void)
+{
+}
+
+void test_close_streams(void)
+{
+}
+
+int test_puts_stdout(const char *str)
+{
+    return write(1, str, strlen(str));
+}
+
+int test_puts_stderr(const char *str)
+{
+    return write(2, str, strlen(str));
+}
+
+static char vprint_buf[10240];
```

Comment:
1. Why move it far from where it's used?  Glad you made that optional ;-)
2. Sure thing.